docs/service-catalog/docs/014-details-etcd-database.md

```diff
@@ -30,79 +27,58 @@ To execute the backup process, you must set the following values in the [core][c
 ### Restore
 
 Follow this instruction to restore an `etcd` cluster from the existing backup.
-Execute all restore commands in the `docs/service-catalog/docs` directory.
-
-> **NOTE:** You must have the backup files created by the CronJob backup application from the previous section.
-
-1. Install the etcd-restore-operator:
-```bash
-kubectl create -f assets/etcd-restore/operator-deploy.yaml
-```
 
-2. Create the EtcdRestore Custom Resource Definition:
-```bash
-kubectl create -f assets/etcd-restore/restore-crd.yaml
-```
-
-3. Export the **ABS_PATH** environment variable with the path to the last successful backup file.
+1. Export the **ABS_PATH** environment variable with the path to the last successful backup file.
 ```bash
 export ABS_PATH=$(kubectl get cm -n kyma-system sc-recorded-etcd-backup-data -o=jsonpath='{.data.abs-backup-file-path-from-last-success}')
+export BACKUP_FILE_NAME=etcd.backup
 ```
 
-> **NOTE:** The ConfigMap name is defined [here][sc-backup-sub-chart] as the **APP_BACKUP_CONFIG_MAP_NAME_FOR_TRACING**.
-
-4. Export the **SECRET_NAME** environment variable with the Secret name to the ABS:
-```bash
-export SECRET_NAME=etcd-backup-abs-credentials
-```
-
-> **NOTE:** The Secret name is defined [here][core-chart-values] under the **global.etcdBackup.secretName** property.
+2. Download the backup to local workstation
```

Comment:
``2. Download the backup to the local workstation. You can do it from the portal or by using [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest). Set the downloaded file path to `$BACKUP_FILE_NAME`.``